internal/templator/templator.go

```diff
@@ -56,6 +57,7 @@ func NewTemplator(box *packr.Box) *Templator {
 		ProtoHealthTemplate:  NewSingleFileTemplator(box, "proto/health_proto.tmpl"),
 		ProtoServiceTemplate: NewSingleFileTemplator(box, "proto/service_proto.tmpl"),
 		Go:                   NewGoTemplator(box),
+		ApiCommit0:           NewSingleFileTemplator(box, "commit0/api_generated.tmpl"),
```

Comment:
The output of the ui process should just be a regular `commit0.yaml` using the same template